tests/queries.py

```diff
@@ -134,6 +137,38 @@ def test_filtering_on_normal_and_eav_fields(self):
         self.assertEqual(p.count(), 2)
 
         # Eugene
-        q1 = Q(name__contains='E', eav__fever=yes)
+        q1 = Q(name__contains='E', eav__fever=self.yes)
         p = Patient.objects.filter(q1)
         self.assertEqual(p.count(), 1)
+
+    def test_order_by(self):
+        def order(ordering):
+            query = Patient.objects.all().order_by(*ordering)
+            return list(query.values_list('name', flat=True))
+
+        self.init_data()
+
+        self.assertEqual(
+            ['Bob', 'Eugene', 'Cyrill', 'Anne', 'Daniel'],
+            order(['eav__city'])
+        )
+
+        self.assertEqual(
+            ['Eugene', 'Anne', 'Daniel', 'Bob', 'Cyrill'],
+            order(['eav__age', 'eav__city'])
+        )
+
+        self.assertEqual(
+            ['Eugene', 'Cyrill', 'Anne', 'Daniel', 'Bob'],
+            order(['eav__fever', 'eav__age'])
+        )
+
+        self.assertEqual(
+            ['Eugene', 'Cyrill', 'Daniel', 'Bob', 'Anne'],
+            order(['eav__fever', '-name'])
+        )
+
+        self.assertEqual(
+            ['Eugene', 'Daniel', 'Cyrill', 'Bob', 'Anne'],
+            order(['-name', 'eav__age'])
+        )
```

Comment:
Continuing my comment above, the test should perhaps include two more scenarios:

`order_by` with just another, non-EAV field with double underscore, and
`order_by` with both non-EAV and EAV double underscore ordering field